content/pt-br/virtual_machine.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Máquina Virtual
+status: Completed
+category: Tecnologia
+---
+
+## O que é
+
+Uma máquina virtual ou VM (do inglês "Virtual Machine") é a inexistência de vínculo de um computador e seu sistema operacional a um hardware específico. As VMs dependem da [virtualização](/pt-br/virtualization/) para transformar um único computador físico em vários computadores virtuais. Essa separação permite que organizações e provedores de infraestrutura criem e destruam facilmente VMs sem afetar o hardware fisicamente.
+
+## Problema relacionado
+
+As máquinas virtuais aproveitam da virtualização. Quando uma máquina [bare metal](/pt-br/bare_metal_machine/) está vinculada a um único sistema operacional, os recursos da máquina que podem ser usados é um pouco limitado. Além disso, quando um sistema operacional está vinculado a uma única máquina física, sua disponibilidade está diretamente ligada a esse hardware. Se a máquina física estiver desligada devido a falhas de manutenção ou hardware, o sistema operacional também estará.
```

Comment:
`Os recursos da máquina que podem ser usados são um pouco limitados.`

Reply:
Sorry about that.

Reply:
No worries, thanks!